charts/application/Chart.yaml

```
@@ -7,4 +7,4 @@ maintainers:
   - name: MediaMarktSaturn
     url: https://github.com/MediaMarktSaturn
 appVersion: 1.0.0
-version: 1.20.1
+version: 1.20.2
```

```suggestion
version: 1.21.0
```
would say, that's a feature and not a bug ;)